src/clocksi_readitem.erl

```diff
@@ -68,6 +71,23 @@ async_read_data_item({Partition, Node}, Key, Type, Transaction, PropertyList, {f
     } end),
     ok.
 
+-spec validate_or_read_data_item(index_node(), key(), type(), object_token(), tx(), read_property_list()) -> {ok, {invalid, snapshot(), object_token()}} | {ok, valid} | {error, term()}.
+validate_or_read_data_item({Partition, Node}, Key, Type, Token, Transaction, PropertyList) ->
```

Comment:
We might want to remove this, it is only used for mock tests in the `interactive_coord_clocksi`. I wanted to mimick what exists for the read API.

Reply:
You could move it into the TEST part, then it is only available when compiled for testing. But I am ok with keeping it similar to the read API.